client/components/ItemDetails.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class ItemDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <ul>
+          {this.props.details.map((item, key) => {
+            return (<li key={key}>{item}</li>)
+          })}
```

Comment:
Can you pass in two parameters to a map function like this?

Reply:
Yes. Map has an optional second parameter which is the index of the item in the array that is being mapped.

Reply:
It also has a third optional parameter which is the entire array.

Reply:
Sure enough. I feel kind of silly now. :)